### PR TITLE
feat: [] add home location example app

### PR DESCRIPTION
**Pull request by loweisz**
Adds a simple app for the home location that shows some stats of the current space

Screen Shot 2022-08-10 at 16 20 24

### CODE REVIEW

File: examples/home-location/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+This project was bootstrapped with [Create Contentful App](https://github.com/contentful/create-contentful-app).
```
Comment (shikaan) [inline examples/home-location/README.md:1]: Should we add a readme with the specifics of this app?

**Review by shikaan — COMMENTED** [review]
Looks good, we can just a little more info in the README

**Comment by colomolo** [comment]
Shouldn't we use npm for all examples?

**Review by colomolo — APPROVED** [review]
lgtm

**Review by shikaan — APPROVED** [review]

**Pull request merged**